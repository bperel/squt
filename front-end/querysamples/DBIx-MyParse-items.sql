SELECT LEFT(field, 6) = 'string' AS alias
