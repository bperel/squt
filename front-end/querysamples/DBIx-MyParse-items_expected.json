{
   "Tables" : {
      "" : {
         "" : {
            "OUTPUT" : {
               "field" : {
                  "1" : "field"
               }
            }
         }
      }
   },
   "Functions" : {
      "1" : {
         "to" : "alias",
         "Constants" : {
            "6" : "6"
         },
         "name" : "left",
         "alias" : "1"
      },
      "alias" : {
         "to" : "OUTPUT",
         "Constants" : {
            "string" : "string"
         },
         "name" : "=",
         "alias" : "alias"
      }
   }
}
